### PR TITLE
Add support to apply (binary) patches after main CUDA install

### PR DESCRIPTION
**Pull request by wpoely86**
The make binary patches instead of updating the version: https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=CentOS&target_version=7&target_type=runfilelocal

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 Do you have an example easyconfig for this?

**Comment by boegel** [comment]
Ah, nevermind, https://github.com/easybuilders/easybuild-easyconfigs/pull/6754 needs it.

**Comment by damianam** [comment]
lgtm!

File: easybuild/easyblocks/c/cuda.py [hunk 1]
```diff
@@ -124,6 +133,11 @@ def create_wrapper(wrapper_name, wrapper_comp):
 
     def sanity_check_step(self):
         """Custom sanity check for CUDA."""
+
+        versionfile = read_file(os.path.join(self.installdir, "version.txt"))
```
Comment (boegel) [inline easybuild/easyblocks/c/cuda.py]: @wpoely86 At the very least, this should be conditional based on the CUDA version. This `version.txt` file is not there for CUDA 7.0.28 for example, so this change is not backward-compatible...

Also, it really only makes sense to check when patches are involved, i.e. when `len(self.src) > 1`?

Comment (boegel) [reply easybuild/easyblocks/c/cuda.py]: fixed in #1483

**Pull request merged**